test/runtests.jl

```diff
@@ -1,5 +1,5 @@
 using OptimalTransport
-using Pkg: Pkg
+using Pkg
```

Comment:
I think this change is not needed?
```suggestion
using Pkg: Pkg
```

Reply:
as above. something odd happening with the CI